.github/labeler.yml

```diff
@@ -1,3 +1,9 @@
+"c api":
+  - changed-files:
+    - any-glob-to-any-file: "src/lib*-c/**/*"
```

Comment:
Maybe it's less noisy of we don't match the implementation part, but we'll see how often those are touched in practice and whether we want the label on such PRs. For instance, if the C++ part changes, the C API isn't really the topic.
We'll see.